cmd/collector/app/server/grpc.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2020 The Jaeger Authors.
```

Comment:
This file seems to be replacing `cmd/collector/app/grpcserver/grpc_server.go`, which was deleted (git didn't recognize it as a move).

Reply:
Did you mean to set the copyright headers to `// Copyright (c) 2018 The Jaeger Authors.` here, as it was in the `grpc_server.go` file?